src/components/ui/login/Login.tsx

```diff
@@ -4,27 +4,31 @@ import {FC, useState} from 'react'
 import Button from "@/components/ui/button/Button";
 import GoogleLogo from "@/assets/svgPaths/googleSVG";
 import {Utils} from "@/lib/utils";
-import {Icons} from "@/components/Icons";
+import {MessagesSquare} from "lucide-react";
+import Image from "next/image";
+import Link from "next/link";
 
-const Login:FC = () => {
+const Login:FC =()=>{
     const [isLoading, setIsLoading] = useState<boolean>(false)
-    const Logo = Icons.Logo
-    return <div className= 'login-page'>
-            <div className='login-body'>
-                <div>
-                    <Logo className='login-logo'/>
-                    <h2 className='login-h2'>
-                        Sign in with Google
-                    </h2>
+    return <div className='p-5 justify-items-center'>
+        <h1 className='flex'>Wirt's Realtime Chat<MessagesSquare/></h1>
+        <div className='md:flex'>
+            <Image alt="Wirt Salthouse" width={275} height={275} src="/wirt_salthouse.jpg" className='rounded-sm'/>
```

Comment:
encapsulate into own component: don't like having the width and height as magic numbers